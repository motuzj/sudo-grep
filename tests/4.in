multiple words search + ignore case
-i "ipsum dolor"
