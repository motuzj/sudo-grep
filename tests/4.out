Lorem ipsum DoloR sit amet, 
Lectus ipsum dOLor purus non. 
